neuralmonkey/encoders/attentive.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=too-few-public-methods
+class Attentive(object):
+    def __init__(self, attention_type, **kwargs):
+        self._attention_type = attention_type
+        self._attention_kwargs = kwargs
+
+        assert hasattr(self, "name")
+        assert hasattr(self, "_padding")
+        assert hasattr(self, "_attention_tensor")
```

Comment:
Mělo by to být v konstruktoru kvůli tomu, jak se tvoří tensorflow graf z konfiguráku - neni tam moc prostoru pro volání dalších metod..
Ale mohl by to volat dekodér, kterej už dostává hotový enkodéry, to je fakt. To by pak i znamenalo, že pokud attention nepoužijeme, tak by se tahle část grafu v enkodéru ani zbytečně nevytvářela.